arlo_server/__init__.py

```diff
@@ -71,12 +73,17 @@ def contest_status(election):
     contests = {}
 
     for contest in election.contests:
-        contests[contest.id] = dict(
-            [[choice.id, choice.num_votes] for choice in contest.choices]
-        )
-        contests[contest.id]["ballots"] = contest.total_ballots_cast
-        contests[contest.id]["numWinners"] = contest.num_winners
-        contests[contest.id]["votesAllowed"] = contest.votes_allowed
+        name = contest.id
+        info_dict = {
+            "ballots": contest.total_ballots_cast,
+            "numWinners": contest.num_winners,
+            "votesAllowed": contest.votes_allowed,
+        }
+
+        for choice in contest.choices:
+            info_dict[choice.id] = choice.num_votes
```

Comment:
Why do we make the candidates top level fields? Didn't that lead to that bug before where the code thought `votesAllowed` was a candidate?

Any reason not to just pass in the original `contest` object to the `Contest` constructor?

Reply:
Hmm, you're right re: candidates. That was a quickly written part of the code, so we didn't anticipate these kinds of problems. 

We could pass in the models.contest object, I just didn't /shrug

Since I merged this, that could be another PR.